project/src/main/java/com/example/javadb/model/Resource.java

```diff
@@ -0,0 +1,144 @@
+package com.example.javadb.model;
+
+import jakarta.persistence.*;
```

Comment:
May need to specify each import because I don't think the * will pass style checks

Reply:
Yep, thanks for catching